super_gui/src/components/Activeparkpass.tsx

```diff
@@ -84,6 +84,7 @@ const Activeparkpass = () => {
         console.log(`Parking Code:`, data);
       } else {
         console.error("Failed to create parking code");
+        toast.error("Only 4 passes allowed per tenant.");
```

Comment:
you could add a if statement that checks for the error code. Now all the errors will resolve in this message, which is not helpful in all circustances. For example, a 500 error code is a internal server error. This does not per se mean that it depends on the number of parking codes.